Examples/PcapPrinter/main.cpp

```diff
@@ -116,6 +118,12 @@ std::string printFileSummary(pcpp::IFileReaderDevice* reader)
 		pcpp::LinkLayerType linkLayer = pcapReader->getLinkLayerType();
 		stream << "   Link layer type: " << linkLayerToString(linkLayer) << std::endl;
 	}
+	else if (dynamic_cast<pcpp::SnoopFileReaderDevice*>(reader) != NULL)

```

Comment:
We can probably add a test for a snoop file in [`test_pcapprinter.py`](https://github.com/seladb/PcapPlusPlus/blob/master/Tests/ExamplesTest/tests/test_pcapprinter.py)

Reply:
Added testcase in 27c7d04